### ISSUE TITLE
Create SuperAdmin role

### ISSUE THREAD

**Issue by patriciahswe** [psu-libraries/scholarsphere-3#243]
We need a role in ScholarSphere that has the following privileges:
1. Ability to remove any user's files 
2. Ability to change visibility / access of a user's files (e.g., make a file private)
3. Ability to correct a file's metadata

**Comment by carolyncole** [comment]
@patriciahswe Can we clarify remove.  Is that just making a file private, or actually deleting the file?  Also where would you expect to perform the action?  When you are on the edit page of the paper, or are you expecting to see all the user's files in your dashboard.

@patriciahswe do 2 & 3 just boil down to edit access to all files metadata and access information, or was there something you wanted to be able to block?

**Comment by patriciahswe** [comment]
Yes, we can clarify! Let's rephrase to "Ability to make a file private." 

That's an interesting albeit overwhelming concept - the ability to see all users' files in the SuperAdmin's dashboard. I'd be interested in seeing all new files uploaded appear in the SuperAdmin's dashboard and having them remain in the dashboard for a limited amount of time - during which the SuperAdmin could review the files and, as necessary, make changes to the metadata or make the files private until further notice.

Re: 2 and 3 - That seems reasonable: access to metadata for all files and access to visibility / access information.

Does the above make sense?

**Comment by carolyncole** [comment]
Ok so I am seeing three user stories:

As a system administrator I would like to be able to modify permissions on all files so that I can restrict access to private for invalid data.

As a system administrator I would like to be able to modify metadata on all files and collections so that we can curate metadata entered into ScholarSphere.

As a system administrator I would like a dashboard that shows me all files and collections uploaded within the past N(?) days so that I have easy access to new files for curation purposes.

The question in my mind is how you are planning to access the files in stories 1 & 2.  If a file is open access or new to the system then you would see it in search or the Admin dashboard.  If the file is private then it will not show up in the catalog.  Do you want the search to return all files no matter the permissions, or are you thinking you will have a direct link to the data?

**Comment by carolyncole** [comment]
Allow a Super Admin to create collections from other peoples work.

**Comment by carolyncole** [comment]
Miss spelling of a name
Metadata errors depositor is not the creator
Possible to make files the files private

**Comment by carolyncole** [comment]
Mid May for access to edit Public Metadata.
